templates/account/README.md

```diff
@@ -1,163 +1,73 @@
-
-# template: ovc.account
+# template: account
 
 ## Description:
-This actor template is responsible for creating an account on any openVCloud environment.
+This template is responsible for creating an account on any openVCloud environment.
 
 ## Schema:
 
-- description: Arbitrary description of the account. **Optional**
-
-- openvcloud: Name of the g8client used to connect to the environment.
-
-- users: List of vcdusers that will be authorized on the account.
-
+- description: (optional) Arbitrary description of the account.
+- openvcloud: Name of the [openvcloud](../openvcloud) instance used to connect to the environment. If not provided and there is exactly one openvcloud instance configured, this instance will be used (and remembered), otherwise it's an error.
+- users: List of [vcd users](#vdc-user) that will be authorized on the account.
 - accountID: The ID of the account. **Filled in automatically, don't specify it in the blueprint**
-
 - maxMemoryCapacity: The limit on the memory capacity that can be used by the account. Default: -1 (unlimited)
-
 - maxCPUCapacity: The limit on the CPUs that can be used by the account. Default: -1 (unlimited)
-
 - maxNumPublicIP: The limit on the number of public IPs that can be used by the account. Default: -1 (unlimited)
-
 - maxDiskCapacity: The limit on the disk capacity that can be used by the account. Default: -1 (unlimited)
-
 - consumptionFrom: determines the start date of the required period to fetch the account consumption info from. If left empty will be creation time of the account.
-
 - consumptionTo: determines the end date of the required period to fetch the account consumption info from. If left empty will be consumptionfrom + 1 hour.
-
 - consumptionData: consumption data will be saved here as series of bytes which represents a zip file. Example of writing the data:
-```
-service = response.json() # response is the service view that can be obtained from getServiceByName
-with open('/tmp/account.zip', 'wb') as f:
-    f.write(service['data']['consumptionData'])
-```
+
+## Vdc User
+- name: name of the [vdcuser](../vdcuser)
+- accesstype: access type (check OVC documentation for supported access types)
+
 
 ## Add/remove/update user
 
-Use the accountusers parameter to specify the user access right to the account. Note that if only name exist in the entry(no accesstype) then the access right will be by default `ACDRUX`.
+Use the accountusers parameter to specify the user access right to the account. Note that if only name exist in the entry (no accesstype) then the access right will be by default `ACDRUX`.
```

Comment:
I don't agree with this full paragraph and the concept of add/remove/update user from a blueprint.
This force the blueprint to contains the state of the account, which is the not what a blueprint should be used for.

So IMO users modifcation should not be allowed through `update_data`, but instead just exposing some method to add/remove/update a user on the service, and these method can be called in a blueprint.

Reply:
I understand your concern, but this also makes me wonder what exactly the point of `update_data` ? since all it's operation can be done via other actions. May be it's a good idea to drop support for update_data method, which will force the developer in thinking differently regarding update operations.